src/lib.rs

```diff
@@ -649,6 +648,21 @@ impl dyn TestStatus {
     }
 }
 
+fn retry_with_backoff<T>(mut f: impl FnMut() -> io::Result<T>) -> io::Result<T> {
+    let mut res = f();
+    for i in 0..10 {
+        if let Err(err) = &res {
+            if let io::ErrorKind::WouldBlock = err.kind() {
+                std::thread::sleep(Duration::from_millis(100 * i));
```

Comment:
```suggestion
                std::thread::sleep(Duration::from_millis(100 * i + 10));
```
Let's sleep a bit also after the very first attempt?

Reply:
oh 😆 good point